README.md

```diff
@@ -254,6 +254,26 @@ This enables the business logic, written and tested in Scala, to be utilised on
 
 Once the configuration is loaded the JS interface is no longer dependent on the microservice and will continue to run inside the users browser should the microservice be shut down.
 
+## Maintaining Libraries and Reproducible Builds
+
+
+### Building and Assembling the Project:
+- When building and assembling the project, we solely rely on the provided package-lock.json.
+- This means using the npm ci command instead of npm install.
+
+### npm ci:
+
+- Installs dependencies based on the package-lock.json file.
+- Ensures that the exact versions specified in package-lock.json are installed. 
+
```

Comment:
can we update the steps to make it run on local. Like first we have to do sbt dist and then sbt run or npm start so everyone trying to work here can easily make it run on local ?